CONTRIBUTING.md

```diff
@@ -1,113 +1,29 @@
 
-The documentation is built using [Sphinx](http://sphinx-doc.org) and [reStructuredText](http://sphinx-doc.org/rest.html), and then hosted by [ReadTheDocs](http://aspnet.readthedocs.org).
+## Contributing process
 
-## Video: Getting Started ##
-[Watch a video](http://ardalis.com/contributing-to-asp-net-5-documentation) showing how to get started building the documentation locally.
+Small changes to one *.MD* file can be made in the browser by tapping the **Edit on GitHub** link in the upper right corner of the browser window. For new articles and changes involving more than one file:
 
-## Building the Docs ##
+* Open an issue describing the article you wish to write or update. Get approval before you invest too much time.
+* Fork the [aspnet/docs](https://github.com/aspnet/Docs/) repo and create a branch for your article.
+* Write or update your article using [DocFX](https://dotnet.github.io/docfx/tutorial/docfx_getting_started.html#2-use-docfx-as-a-command-line-tool) in [GitHub Flavored Markdown](https://guides.github.com/features/mastering-markdown/). Images go in the *\<article-name>/_static* folder. Code goes in the *\<article-name>/samples* folder. It's often helpful to review an existing article to use as an template (for example, *tutorials\first-web-api.md* and the assets in *tutorials\first-web-api*).
+* Submit a pull request (PR) from your branch to `aspnet/docs/master` and respond to PR feedback.
 
-Once you have cloned the Docs to your local machine, the following instructions will walk you through installing the tools necessary to build and test.
+## Building the docs
 
-1. [Download python](https://www.python.org/downloads/) version 2.7.10 or higher (Version 3.5 is recommended). If you install 3.5, check the include python in path option.
+* Download and unzip *docfx.zip* from [DocFX releases](https://github.com/dotnet/docfx/releases). Add DocFX to your PATH.
+* Build and preview your changes with [DocFX command line](https://dotnet.github.io/docfx/tutorial/docfx_getting_started.html#2-use-docfx-as-a-command-line-tool). Navigate to the aspnet folder (which contains the *docfx.json* file) and enter the following in a command prompt:
 
-2. Skip this step if you installed python 3.5 and checked the "include in python path" option. If you are installing on Windows, ensure both the Python install directory and the Python scripts directory have been added to your `PATH` environment variable. For example, if you install Python into the c:\python34 directory, you would add `c:\python34;c:\python34\scripts` to your `PATH` environment variable.
-
-3. Clone the docs: 
-
-    ```git clone https://github.com/aspnet/Docs.git```
- 
-4. Navigate to the Docs repo and install the build dependencies:
-
-	```cd docs```
-
-	```pip install -r requirements.txt```
-	
+```
+docfx -t default --serve
```

Comment:
Indent code and following explanation of it so it goes with the bullet text that introduces it.